lib/auth.config.ts

```diff
@@ -1,4 +1,5 @@
 import { NextAuthConfig } from "next-auth";
+import { getUserAvater } from "./db";
```

Comment:
this function is not used, but it shoudnl't need to be if the url is saved in the users1 table rather than the files1 table